app/ts-meta/meta/handler.go

```diff
@@ -411,3 +417,47 @@ func (h *httpHandler) serveExpandGroups(w http.ResponseWriter, r *http.Request)
 		h.logger.Error("write result failed", zap.Error(err))
 	}
 }
+
+// curl -i -XPOST 'http://127.0.0.1:8091/leadershiptransfer'
+func (h *httpHandler) leadershipTransfer(w http.ResponseWriter, r *http.Request) {
+	err := h.store.leadershipTransfer()
+	if err == raft.ErrNotLeader {
+		l := h.store.leader()
+		if l == "" {
+			h.httpErr(errors.New("no raft leader"), w, http.StatusServiceUnavailable)
```

Comment:
Does it mean this is not raft leader and you need execute this command on leader? Please check it and return more exact error message.